database/pgsql/complex_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 clair authors
+// Copyright 2016 clair authors
```

Comment:
Rather than spamming the codebase, I'd rather just update them whenever the files are touched.